Source/Ably.modulemap

```diff
@@ -5,6 +5,7 @@ framework module Ably {
     module * { export * }
 
     explicit module Private {
+        header "ARTConstants.h"
```

Comment:
Why is this listed as private?

Reply:
Because the lib user doesn't need to have access to those constants.